redis/cache/redisx/redisxtest/redisxtest.go

```diff
@@ -0,0 +1,39 @@
+package redisxtest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/joomcode/redispipe/redis"
+	"github.com/joomcode/redispipe/redisconn"
+
+	"github.com/reddit/baseplate.go/redis/cache/redisx"
+)
+
+// NewMockRedisClient sets up a redis client for use in testing
+func NewMockRedisClient(
```

Comment:
now there's nothing really "mocking" here (the mocking part is done via miniredis, which is no longer done here).

I would rename this function to `NewTestRedisClient`, and document that:

1. the `address` arg should be coming from miniredis (we should also provide an example on how to it, see https://pkg.go.dev/testing#hdr-Examples, we also have prior arts in baseplate.go, for example https://github.com/reddit/baseplate.go/blob/master/errorsbp/batch_size_example_test.go)
2. we should also document that the returned client will be auto released at the end of test and there's no need to do that explicitly (because the caller actually still have access to that via `client.SyncCtx.S.Close()`